src/main/java/rapgod/RapGod.java

```diff
@@ -0,0 +1,90 @@
+package rapgod;
+
+import rapgod.exceptions.NoInputException;
+import rapgod.exceptions.RudeInputException;
+import rapgod.ui.EchoBot;
+import rapgod.ui.ListBot;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class RapGod {
+
+    public static Scanner scanner = new Scanner(System.in);
+    public static final ArrayList<String> RUDE_WORDS = new ArrayList<>(Arrays.asList(
+            "damn", "hell", "shit", "fuck",
+            "bitch", "asshole", "dickhead",
+            "idiot", "moron", "stupid",
+            "loser", "jerk", "creep"
+    ));
+
+    private static void run() {
+        String logo = """
+                  ____              ____           _\s
+                |  _ \\ __ _ _ __  / ___| ___   __| |
+                | |_) / _` | '_ \\| |  _ / _ \\ / _` |
+                |  _ < (_| | |_) | |_| | (_) | (_| |
+                |_| \\_\\__,_| .__/ \\____|\\___/ \\__,_|
+                           |_|                     \s
+                """;
+
+        String header = """
+                -----------------------------------------------
+                 I'm beginning to feel like a:
+                """;
+
+        String introduction = """
+                -----------------------------------------------
+                Hello. I'm RapGod
+                What can I do for you today?
+                
+                [OPTIONS]
+                'Echo bot' to echo
+                'List bot' to create a list
+                'Bye' to exit
+                -----------------------------------------------
+                """;
+
+        System.out.println(header);
+        System.out.println(logo);
+        System.out.print(introduction);
+
+        while(true) {
```

Comment:
No space between while(true).
```suggestion
        while (true) {
```